Jenkinsfile.release

```diff
@@ -30,9 +30,6 @@ node ('macos1'){
       slackSend color: 'good', message: BRANCH_NAME + ' build started. ' + env.BUILD_URL
 
       checkout scm
```

Comment:
How does this get release branch?

Reply:
I understand it uses the revision that triggered the pipeline build